src/cockpit/389-console/src/lib/ldap_editor/wizards/operations/addCosDefinition.jsx

```diff
@@ -0,0 +1,1402 @@
+                    {this.state.adding &&
+                        <div>
+                            <Spinner className="ds-left-margin" size="md" />
+                            &nbsp;&nbsp;Adding CoS definition ...
+                        </div>
+                    }
+                </Alert>
+                {resultVariant === 'danger' &&
+                    <Card isHoverable>
+                        <CardTitle>
+                            LDIF Data
+                        </CardTitle>
+                        <CardBody>
+                            {ldifLines.map((line) => (
+                                <h6 key={line.id}>{line.data}</h6>
+                            ))}
+                        </CardBody>
+                    </Card>
+                }
+                <Modal
+                    variant={ModalVariant.small}
+                    title="Create CoS Template"
+                    isOpen={this.state.isConfirmModalOpen}
+                    onClose={this.handleConfirmModalToggle}
+                    actions={[
+                        <Button key="confirm" variant="primary" onClick={this.handleCreateTemplateEnd}>
+                        Create Template
+                        </Button>,
+                        <Button key="cancel" variant="link" onClick={this.handleConfirmModalToggle}>
+                        Cancel
+                        </Button>
+                    ]}
+                    >
+                    <Grid>
+                        <GridItem span={12}>
+                            <TextContent className="ds-margin-top">
+                                <Text>
+                                    You've chosen 'Classic' CoS type.
+                                    cosTemplateDN attribute is set to {this.state.createdDefiniton}.
+                                </Text>
+                                <Text>
+                                    Do you want to create a CoS template now?
+                                </Text>
+                                <Text>
+                                    It will be added as a child to this entry: '{this.state.createdDefiniton}'
+                                </Text>
+                            </TextContent>
+                        </GridItem>
+                    </Grid>
+                </Modal>
+            </div>
+        );
+
+        const addCoSSteps = [
+            {
+                id: 1,
+                name: this.props.firstStep[0].name,
+                component: this.props.firstStep[0].component,
+                canJumpTo: stepIdReached >= 1 && stepIdReached < 9,
+                hideBackButton: true
+            },
+            {
+                id: 2,
+                name: 'Select Type',
+                component: namingValAndTypeStep,
+                enableNext: namingVal === '' ? false : true,
+                canJumpTo: stepIdReached >= 2 && stepIdReached < 9,
+                hideBackButton: true
```

Comment:
This should be allowed to move back.  You can jump to step 1 anyway, but I see no reason to block it here.